File: R/tests/testthat/test_pst_Q.R
context("Test pst_Q")
library(hBayesDM)

test_that("Test pst_Q", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(pst_Q(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_pst_Q.py
import pytest

from hbayesdm.models import pst_Q


def test_pst_Q():
    _ = pst_Q(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
